// react-native-paper-dates/lib/commonjs/Date/DatePickerModalHeaderBackground.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = DatePickerModalHeaderBackground;
var _react = _interopRequireDefault(require("react"));
var _reactNative = require("react-native");
var _utils = require("../shared/utils");
var _reactNativeSafeAreaContext = require("react-native-safe-area-context");
var _jsxRuntime = require("react/jsx-runtime");
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
function DatePickerModalHeaderBackground({
  children
}) {
  const backgroundColor = (0, _utils.useHeaderBackgroundColor)();
  const insets = (0, _reactNativeSafeAreaContext.useSafeAreaInsets)();
  return /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNative.Animated.View, {
    style: [styles.animated, {
      backgroundColor,
      paddingLeft: insets.left,
      paddingRight: insets.right
    }],
    children: children
  });
}
const styles = _reactNative.StyleSheet.create({
  animated: {
    elevation: 4
  }
});
